app-frontend/packs/components/dashboard/homePage/HomePageView.vue

```diff
@@ -1,19 +1,36 @@
 <template>
 
-    <LayoutPage>
-        <HomePage></HomePage>
-    </LayoutPage>
+    <HomePageEmptyBanner @clickNewBannerEmpty="createNewBanner" v-if="isShowBlock"></HomePageEmptyBanner>
+
+    <HomePageWithBanners @clickNewBanner="createNewBanner" v-else></HomePageWithBanners>
```

Comment:
и по сути переименовав свой ивент, оба компонента будут слушать событие `create` и логически становится понятнее что делает этот ивент  

```suggestion
    <HomePageEmptyBanner @create="createNewBanner" v-if="isShowBlock" />

    <HomePageWithBanners @create="createNewBanner" v-else />

Reply:
исправил